docs/pages/database-access/reference/cli.mdx

```diff
@@ -103,21 +103,26 @@ $ teleport db configure create \
 | - | - |
 | `--proxy` | Teleport Proxy Service address to connect to. Default: `0.0.0.0:3080`. |
 | `--token` | Invitation token to register with the Auth Service. Default: none. |
-| `--rds-discovery` | List of AWS regions in which the Database Service will discover RDS/Aurora instances. |
-| `--redshift-discovery` | List of AWS regions in which the Database Service will discover Redshift instances. |
-| `--elasticache-discovery` | List of AWS regions in which the Database Service will discover ElastiCache Redis clusters. |
-| `--memorydb-discovery` | List of AWS regions in which the Database Service will discover MemoryDB clusters. |
+| `--rds-discovery` | List of AWS regions in which the agent will discover RDS/Aurora instances. |
```

Comment:
Just curious if there's a convention being followed to sort this list?

Reply:
hmm.. i don't know if we have such convention. I think this list should not be sorted. the ordering currently matches the same ordering they appear in the command help (`teleport db configure create -h`).

Reply:
I don't think you need to sort them here. This question is mostly for fact-finding, as the docs team considers how we can improve reference documentation in general to make it easier for users to find what they're looking for.